_includes/radio.html

```diff
@@ -11,7 +11,7 @@ <h1>
     {% for answer in item.answers %}
     <div class="usa-radio">
         <input class="usa-radio__input usa-radio__input--tile" id="{{ item.slug }}{{ forloop.index }}" type="radio" name="{{ item.slug }}" value="{{ answer.path }}">
-        <label class="usa-radio__label" for="{{ item.slug }}{{ forloop.index }}">{{ answer.text }}</label>
+        <label class="usa-radio__label margin-bottom-6" for="{{ item.slug }}{{ forloop.index }}">{{ answer.text }}</label>
```

Comment:
This utility class doesn't seem to have any effect. What are you trying to accomplish?

Reply:
LOL absolutely nothing! I was getting lost in all the files! Thank you for catching this, I'll have to be more careful next time.